src/Raven.Client/Documents/Operations/TimeSeries/TimeSeriesDetails.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace Raven.Client.Documents.Operations.TimeSeries
+{
+    public class TimeSeriesDetails
+    {
+        public string Id;
```

Comment:
convert to properties. It is easier to inherit and override if necessary in the future.

Reply:
done